darts/timeseries.py

```diff
@@ -31,6 +32,9 @@ def __init__(self,
         ----------
         df
             The actual time series, as a pandas DataFrame with a proper time index.
+        columns
+            The columns name in the same order as in the underlying DataFrame. If columns name are not unique they will
```

Comment:
the default behavior is to infer the column names from the dataframe. That's what we get with the `from_dataframe` function but since TimeSeries are immutable, to preserve the column upon operations we need to pass the names as parameter during creation

Reply:
I don't understand this, because when creating the TimeSeries, we are anyway making an internal copy `df_` of the dataframe. So any change of the columns on the original dataframe should not impact the TimeSeries that is already built. However I might miss something?

Reply:
no my bad, the new implementation just keeps the column parameter as a way to override this default behavior of using the underlying DataFrame columns